modules/client/test/mocks/ethers-providers.ts

```diff
@@ -0,0 +1,8 @@
+const original = jest.requireActual("@ethersproject/providers");
+export const mockedEthersProviders = {
+  ...original,
+  getNetwork: jest.fn().mockReturnValue({ chainId: 31337, name: "goerli" }),
```

Comment:
nope this was a file that i created for testing but it ended up not working up well because the JSONRpcProvider detect the network changes and crashes